pyClarion/components/blas.py

```diff
@@ -216,27 +216,14 @@ def step(self):
             self.add(key)
         self._add.clear()
 
-    def prune(self):
+    def prune(self) -> None:
         """Drop all entries that are below threshold."""
 
         below = [key for key, bla in self.items() if bla.below_threshold]
         for key in below:
             del self[key]
 
-    def keys_below_threshold(self, keys):
```

Comment:
`keys_below_threshold()` should not be removed; it gets called in `buffers.py`. I may have mistakenly asked you to delete it when we were merging.